tools/devtools/nodes/inputs.py

```
@@ -347,6 +347,20 @@ def node_with_pre_attach_legacy_widgets(self):
         return ()
 
 
+class NodeWithComparerWidget:
+    @classmethod
+    def INPUT_TYPES(cls):
```

Ruff (0.16.3)

[warning] Missing return type annotation for classmethod `INPUT_TYPES`

(ANN206)